assets/js/base/components/cart-checkout/shipping-calculator/address.tsx

```diff
@@ -39,12 +39,16 @@ const ShippingCalculatorAddress = ( {
 		showAllValidationErrors();
 		return ! hasValidationErrors();
 	};
+	const { clearValidationError } = dispatch( 'wc/store/validation' );
```

Comment:
Since this is a React component we could use `useDispatch` here. `dispatch` should be used outside of the React render tree.